tests/runner.py

```diff
@@ -1,8 +1,8 @@
-from time import strftime
 import subprocess
 from datetime import datetime
 from re import search
-import os
+from time import strftime
```

Comment:
Re-arrange import sequence to match PEP8 (import then from ... import), and remove unused import (`import os`)